test/dlc_tests/conftest.py

```diff
@@ -341,7 +341,7 @@ def delete_ssh_keypair():
                 or "mxnet_inference" in request.fixturenames
             )
         )
-        or (is_neuron_image)
+        or (is_neuron_image(request.fixturenames))
```

Comment:
this was missing causing every execution to mark as true, leading 300g for each instances